static/app/stores/repositoryStore.tsx

```diff
@@ -79,6 +76,10 @@ const storeConfig: Reflux.StoreDefinition & RepositoryStoreInterface = {
   get() {
     return {...this.state};
   },
+
+  getState() {
+    return this.get();
+  },
```

Comment:
Could you open this as a separate PR `ref(js): Make repositoryStore useLegacyStore compatible`?